Tools/LyTestTools/ly_test_tools/launchers/platforms/win/launcher.py

```diff
@@ -169,29 +169,13 @@ def configure_settings(self):
         host_ip = '127.0.0.1'
         self.args.append(f'--regset="/Amazon/AzCore/Bootstrap/project_path={self.workspace.paths.project()}"')
         self.args.append(f'--regset="/Amazon/AzCore/Bootstrap/remote_ip={host_ip}"')
-        self.args.append('--regset="/Amazon/AzCore/Bootstrap/wait_for_connect=1"')
```

Comment:
should this also be removed from the linux/launcher.py in the file above or is this a windows specific difference?

Reply:
Good catch, removed!